mesonbuild/interpreter/primitives/integer.py

```diff
@@ -65,10 +65,13 @@ def is_even_method(self, args: T.List[TYPE_var], kwargs: TYPE_kwargs) -> bool:
     def is_odd_method(self, args: T.List[TYPE_var], kwargs: TYPE_kwargs) -> bool:
         return self.held_object % 2 != 0
 
-    @noKwargs
+    @typed_kwargs(
+        'to_string_method',
+        KwargInfo('padding', types=int, required=False, default=0, since='1.3.0')
```

Comment:
Nitpicking: When you have a default value you don't need `required=False`.

Reply:
Also you don't need `types=`

```suggestion
        KwargInfo('padding', int, default=0, since='1.3.0')
```

Reply:
> 

I tried removing this one, but I get:

`TypeError: KwargInfo.__init__() missing 1 required positional argument: 'types'`